include/endian.h

```diff
@@ -146,12 +146,24 @@
 
 #define betoh16               be16toh
 #define letoh16               le16toh
+#define bemtoh16(x)           htobe16(*(FAR uint16_t *)(x))
```

Comment:
Whyit is not

```suggestion
#define bemtoh16(x)           betoh16(*(FAR uint16_t *)(x))
```
Even it it results to the same it seems to be not intuitive as it is now